web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -24,7 +35,17 @@ function setupMockAnalytics(): typeof analytics {
 describe("<SidebarCardsList />", () => {
   beforeEach(() => {
     jest.resetAllMocks();
-    useMockBusiness({});
+    const business = generateBusiness({
+      profileData: generateProfileData({
+        industryId: "generic",
+        businessPersona: "STARTING",
+        sectorId: undefined,
+      }),
+    });
+    useMockBusiness(business);
+    (useMediaQuery as jest.Mock).mockImplementation(() => {
+      return true;
+    }); // set large screen
```

Comment:
Let's avoid code comments. If something feels like it needs an explanatory comment, then let's write explanatory code instead. For example, a function called `setLargeScreen` here.

Reply:
done